source/jormungandr/jormungandr/tests/authentication_test.py

```diff
@@ -61,3 +61,20 @@ def get_token_basic_auth_unicode_test():
 def get_token_url_test():
     with app.test_request_context('/', query_string='key=mykey'):
         assert get_token() == 'mykey'
+
+def get_used_coverages_test():
+    with app.test_request_context('/v1/coverage/fr-idf'):
+        assert get_used_coverages() == ['fr-idf']
+
+    with app.test_request_context('/v1/coverage/fr-idf'):
+        register_used_coverages(['fr-bre'])
```

Comment:
remark: Don't really see a case were it could happen and why this behaviour is required (why shouldn't we raise in that case?)